torch/optim/adam.py

```diff
@@ -41,6 +42,13 @@ def __init__(self, params, lr=1e-3, betas=(0.9, 0.999), eps=1e-8,
                         weight_decay=weight_decay, amsgrad=amsgrad)
         super(Adam, self).__init__(params, defaults)
 
+        # https://github.com/pytorch/pytorch/pull/22628
+        warnings.warn(
```

Comment:
this will spit out a warning to all Adam users, maybe not worth it?

Reply:
*too noisy

Reply:
Yes, I agree this is too noisy.